packages/Tag/src/Tag.js

```diff
@@ -1,28 +1,38 @@
 import React from "react";
 import PropTypes from "prop-types";
 import RawButton from "@paprika/raw-button";
+import useI18n from "@paprika/l10n/lib/useI18n";
 import * as sc from "./Tag.styles";
 import * as types from "./types";
 
 export default function Tag(props) {
   const { children, onRemove, size, onClick, a11yText, avatar, isDisabled, ...moreProps } = props;
+  const I18n = useI18n();
 
   const handleRemove = e => {
     onRemove();
     e.stopPropagation();
   };
 
+  const handleKeyDown = e => {
+    if (e.key === "Enter" || e.key === "Backspace" || e.key === " ") {
+      handleRemove(e);
+    }
+  };
+
   const content = (
     <>
       {avatar}
       <sc.Ellipsis size={size}>{children}</sc.Ellipsis>
       {onRemove ? (
         <sc.Delete
+          a11yText={I18n.t("remove_a11y", { value: children })}
```

Comment:
What if `children` is a `node`, not a `string`?
Maybe it should fall back to the generic "remove"?